metaflow/plugins/catch_decorator.py

```diff
@@ -21,30 +21,21 @@ def __init__(self, retry_count):
 
 class CatchDecorator(StepDecorator):
     """
-    Step decorator to specify error handling for your step.
+    Specifies that the step will success under all circumstances.
 
-    This decorator indicates that exceptions in the step should be caught and not fail the entire
-    flow.
-
-    This can be used in conjunction with the @retry decorator. In that case, catch will only
-    activate if all retries fail and will catch the last exception thrown by the last retry.
-
-    To use, annotate your step as follows:
-    ```
-    @catch(var='foo')
-    @step
-    def myStep(self):
-        ...
-    ```
+    The decorator will create an optional artifact, specified by `var`, which
+    contains the exception raised. You can use it to detect the presence
+    of errors, indicating that all happy-path artifacts produced by the step
+    are missing.
 
     Parameters
     ----------
     var : string
-        Name of the artifact in which to store the caught exception. If not specified,
-        the exception is not stored
+        Name of the artifact in which to store the caught exception.
+        If not specified, the exception is not stored.
     print_exception : bool
-        Determines whether or not the exception is printed to stdout when caught. Defaults
-        to True
+        Determines whether or not the exception is printed to
+        stdout when caught (Default: True).
```

Comment:
nit: default (that's what you have been using till now)

Reply:
fixed